src/CfgEditor/CfgHtmlBuilder.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+import {getNonce} from '../Utils/external/Nonce';
+import {getUri} from '../Utils/external/Uri';
+
+
+export class CfgHtmlBuilder {
+  webview: vscode.Webview;
+  extensionUri: vscode.Uri;
+
+  constructor(webview: vscode.Webview, extensionUri: vscode.Uri) {
+    this.webview = webview;
+    this.extensionUri = extensionUri;
+  }
+
+  async build(): Promise<string> {
```

Comment:
This function is changed to return `Promise<string>`, not `string`.
But I cannot find anywhere to use `Promise`.
Is there any reason to return `Promise` even it is not used?

Reply:
It's because this function is `async`. And the reason that uses `async` is using `await this.getHtml()`.